packages/storybook/package.json

```diff
@@ -38,18 +38,20 @@
     "@tilburg/web-components-stencil": "workspace:*",
     "@types/react": "18.3.3",
     "@types/react-dom": "18.3.0",
-    "@utrecht/component-library-react": "3.0.1-alpha.41",
```

Comment:
Deze change heeft impact op de look-and-feel van de storybook, het was al niet echt top qua look and feel, wat mij betreft gaan we dat gewoon fixen na het mergen van deze PR